modules/n1ql/pages/n1ql-language-reference/selectclause.adoc

```diff
@@ -295,21 +301,20 @@ ORDER BY city LIMIT 5;
 |===
 ====
 
-=== Keyspace Name
+[[keyspace-name]]
+=== Star Expression ({asterisk})
 
-(Optional; if the keyspace is unspecified in the SELECT clause, it will be taken from the xref:n1ql-language-reference/from.adoc[FROM clause].)
+The star expression `{asterisk}` enables you to select _all_ the fields from the source specified by the xref:n1ql-language-reference/from.adoc[FROM clause].
 
-The name of the keyspace or keyspaces used, separated by a comma.
-
-Keyspaces are equivalent to collections in Couchbase Server.
-A keyspace is a set of documents that may vary in structure, and is a unit of authorization and resource allocation.
+The star expression may be preceded by a xref:n1ql:n1ql-language-reference/nestedops.adoc[path], to select all the nested fields from within an array.
 
 [NOTE]
 ====
-When specifying the keyspace name in `SELECT keyspace_name.*` for all fields, the keyspace name will not appear in the result set; whereas not specifying the keyspace name in `SELECT * FROM keyspace_name` adds the keyspace name to the result set.
+Not including the keyspace name before a star expression or field expression, as in <<q3>>, adds the keyspace name to the result set; whereas if you include the keyspace name before a star expression or field expression, as in <<q4>> and <<q5>>, the keyspace name will not appear in the result set.
```

Comment:
```suggestion
Excluding the keyspace name before a star expression or field expression, as in <<q3>>, adds the keyspace name to the result set; whereas if you include the keyspace name before a star expression or field expression, as in <<q4>> and <<q5>>, the keyspace name will not appear in the result set.
```

Reply:
Not a huge deal.

Comment will probably apply to the whole set.

Reply:
Good call, did something similar